app/uk/gov/hmrc/transitmovementsauditing/services/AuditService.scala

```diff
@@ -50,10 +62,41 @@ trait AuditService {
   def send(auditType: AuditType, jsonStream: Payload)(implicit
     hc: HeaderCarrier
   ): EitherT[Future, AuditError, Unit]
+
+  def getAdditionalField(name: String, path: Seq[String], src: Source[ByteString, _]): Future[ParseResult[(String, String)]]
+
+  def getAdditionalFields(messageType: Option[MessageType], src: Source[ByteString, _]): EitherT[Future, ParseError, (String, String)]
```

Comment:
This will return a single pair, it needs to return a `Seq` of pairs as there will be multiple fields.

Reply:
Indeed. Will change.